src/Platform/Microsoft.Testing.Platform/Builder/TestApplicationBuilder.cs

```diff
@@ -58,9 +58,11 @@ internal TestApplicationBuilder(
 
     internal ITestHostOrchestratorManager TestHostControllersManager => _testHostBuilder.TestHostOrchestratorManager;
```

Comment:
Unrelated to your changes, but all other properties of type "SomethingManager" are named "Something" without "Manager". This is currently internal so not a big deal, but may be worth renaming in a separate PR so that we don't unintentionally end up with naming inconsistency if we decided to make it public in future.